src/shared/lib/FormatUtils.ts

```diff
@@ -13,3 +15,33 @@ export function toPrecision(value:number, precision:number, threshold:number)
 
     return ret;
 }
+
+/**
+ * Works the same as the default toFixed() function when the result string is not expected to be a zero value
+ * such as 0.0, 0.00, 0.000, etc.
+ *
+ * If the default toFixed() function returns a zero value, then converts it to an inequality such as
+ * <0.1, <0.01, <0.001, etc.
+ */
+export function toFixedWithThreshold(value: number, digits: number): string
```

Comment:
`0.1` is dynamic actually. If the `digits` is 1 then it returns `<0.1`, so it is still "fixed" to one digit. If it is 2, returns `<0.01`, etc. This way, we still respect the `digits` parameter.